lib/sending_text.rb

```diff
@@ -0,0 +1,20 @@
+require 'twilio-ruby'
+require 'rubygems'
+
+def sms
+  account_sid = ENV[TWILIO_ACCOUNT_SID]
+  auth_token = ENV[AUTH_TOKEN]
+  num = ENV[NUM]
+
+  @client = Twilio::REST::Client.new(account_sid, auth_token)
+
+  message = @client.messages
+    .create (
+    body: "Thank you, your order has been placed and will arrive at #{(Time.new + 3600).strftime("%H:%M")}."
+    from: '+12513571363'
+    to: num
+  )
+
+  puts message.sid
+  
+end
```

Comment:
It might be interesting to check out https://github.com/makersacademy/takeaway-challenge/blob/master/docs/review.md#stubbing-the-twilio-api as an idea of how you could test this method.